### PR TITLE
Added alt tags for ChRIS website images wherever required

### LINKED ISSUE #23: ALT tags needed for ChRIS website images
Accessibility:
The following images to do not have an ALT tag to provide a screen reader information about the image:
Image without ALT attribute Found on page
https://chrisproject.org/assets/images/illus_collaborate.png	/about
https://chrisproject.org/assets/images/icons/role-docs.png	/join-us
https://chrisproject.org/assets/images/illus_data.png	/about
https://chrisproject.org/assets/images/demo_feed.jpg	/demo
https://chrisproject.org/assets/images/icons/role-marketing.png	/join-us
https://chrisproject.org/assets/images/icons/role-design.png	/join-us
https://chrisproject.org/assets/images/icons/role-backend.png	/join-us
https://chrisproject.org/assets/images/icons/tech-openstack.png	/join-us
https://chrisproject.org/assets/images/thumb_chris-store.jpg	/
https://chrisproject.org/assets/images/join.png	/join-us
https://chrisproject.org/assets/images/icons/tech-django.png	/join-us
https://chrisproject.org/assets/images/icons/tech-kube.png	/join-us
https://chrisproject.org/assets/images/icons/role-qa.png	/join-us
https://chrisproject.org/assets/images/demo_login.jpg	/demo
https://chrisproject.org/assets/images/thumb_creating-chris.jpg	/
https://chrisproject.org/assets/images/demo_output.jpg	/demo
https://chrisproject.org/assets/images/illus_compute.png	/about
https://chrisproject.org/assets/images/demo_feeds.jpg	/demo
https://chrisproject.org/assets/images/icons/tech-python.png	/join-us
https://chrisproject.org/assets/images/icons/role-frontend.png	/join-us
https://chrisproject.org/assets/images/icons/tech-react.png	/join-us
https://chrisproject.org/assets/images/icons/tech-js.png	/join-us

see https://github.com/FNNDSC/website/issues/20

### PR DESCRIPTION
**Pull request by archijaiswal**
Added alt tags for ChRIS website images at possibly all the places it's required.
Fixes #23 
A part of [Issue #20]

### CODE REVIEW

**Comment by archijaiswal** [comment]
Kindly review this and suggest changes.@jennydaman @mairin

File: _pages/for-devs.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ permalink: /for-devs
 </div>
 <div class="row no-gutters position-relative">
     <div class="col-md-4 mb-md-0 p-md-4">
-        <img src="/assets/images/library/thumb-designassets.png" class="w-100" alt="...">
+        <img src="/assets/images/library/thumb-designassets.png" class="w-100" alt="ChRIS design assets graphic.">
```
Comment (jennydaman) [inline _pages/for-devs.md]: Nitpick: would you remove the trailing period? None of the other alt tags are complete sentences so let's try to be consistent.

File: _pages/for-users.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Here's a quick tour of what you'll see in the demo:
 
   <div class="card">
 	  <div class="card-header">
-	    <img style="height: 100%; width: 100%;" src="/assets/images/demo_login.png" />
+	    <img style="height: 100%; width: 100%;" src="/assets/images/demo_login.png" alt="screenshot of the ChRIS web application showing a demo login"/>
```
Comment (jennydaman) [inline _pages/for-users.md]: Another nitpick: every other alt has its first letter capitalized, so let's do the same here.

**Review by jennydaman — CHANGES_REQUESTED** [review]

File: _pages/for-users.md (hunk 2 above)
Comment (archijaiswal) [reply _pages/for-users.md]: Sure, okay!

File: _pages/for-devs.md (hunk 1 above)
Comment (archijaiswal) [reply _pages/for-devs.md]: Thanks, sure!

**Comment by archijaiswal** [comment]
Please let me know if there are any other changes. Thanks for your kind review! @jennydaman

**Review by jennydaman — APPROVED** [review]

**Comment by jennydaman** [comment]
lgtm thanks @archijaiswal !

**Comment by archijaiswal** [comment]
You're most welcome !!@jennydaman

**Pull request merged**